README.md

```diff
@@ -0,0 +1,11 @@
+# com.obsproject.Studio.Plugin.GStreamerVaapi
+Pull requests for new applications to be added
+
+To install and test this OBS plugin simply run:
+```
+flatpak install --user https://dl.flathub.org/build-repo/2496/com.obsproject.Studio.Plugin.Vaapi.flatpakref
```

Comment:
Here too

Reply:
Do I change it to as if it is already on flathub? as in 
```
flatpak install com.obsproject.Studio.Plugin.GStreamerVaapi
```
?

Reply:
Sounds good to me! Please change to that